crypto/evp_extra/evp_test.cc

```diff
@@ -1468,6 +1468,21 @@ TEST(EVPTest, ECTLSEncodedPoint) {
     ERR_clear_error();
 }
 
+TEST(EVPTest, PKEY_set_type_str) {
+  EVP_PKEY *pkey = NULL;
+  pkey = EVP_PKEY_new();
```

Comment:
should wrap this in a `bssl::UniquePtr` to avoid memory leaks in testing.

Reply:
+1